`sbt package` failing on M1 macOS

Running `sbt package` on a fresh clone on an M1 macOS[*1] fails with the following message:

[*1] Darwin Kernel Version 21.5.0: Tue Apr 26 21:08:37 PDT 2022; root:xnu-8020.121.3~4/RELEASE_ARM64_T6000

```
...
[error] java.lang.RuntimeException: error occurred while compiling protobuf files: https://repo.maven.apache.org/maven2/com/google/protobuf/protoc/3.10.0/protoc-3.10.0-osx-aarch_64.exe
[error]         at sbtprotobuf.ScopedProtobufPlugin.executeProtoc(ProtobufPlugin.scala:112)
[error]         at sbtprotobuf.ScopedProtobufPlugin.compile(ProtobufPlugin.scala:128)
[error]         at sbtprotobuf.ScopedProtobufPlugin.$anonfun$sourceGeneratorTask$3(ProtobufPlugin.scala:170)
[error]         at sbt.util.FileFunction$.$anonfun$cached$1(FileFunction.scala:80)
[error]         at sbt.util.FileFunction$.$anonfun$cached$4(FileFunction.scala:153)
[error]         at sbt.util.Difference.apply(Tracked.scala:414)
[error]         at sbt.util.Difference.apply(Tracked.scala:394)
[error]         at sbt.util.FileFunction$.$anonfun$cached$3(FileFunction.scala:149)
[error]         at sbt.util.Difference.apply(Tracked.scala:414)
[error]         at sbt.util.Difference.apply(Tracked.scala:389)
[error]         at sbt.util.FileFunction$.$anonfun$cached$2(FileFunction.scala:148)
[error]         at sbtprotobuf.ScopedProtobufPlugin.$anonfun$sourceGeneratorTask$1(ProtobufPlugin.scala:172)
[error]         at scala.Function1.$anonfun$compose$1(Function1.scala:49)
[error]         at sbt.internal.util.$tilde$greater.$anonfun$$u2219$1(TypeFunctions.scala:62)
[error]         at sbt.std.Transform$$anon$4.work(Transform.scala:68)
[error]         at sbt.Execute.$anonfun$submit$2(Execute.scala:282)
[error]         at sbt.internal.util.ErrorHandling$.wideConvert(ErrorHandling.scala:23)
[error]         at sbt.Execute.work(Execute.scala:291)
[error]         at sbt.Execute.$anonfun$submit$1(Execute.scala:282)
[error]         at sbt.ConcurrentRestrictions$$anon$4.$anonfun$submitValid$1(ConcurrentRestrictions.scala:265)
[error]         at sbt.CompletionService$$anon$2.call(CompletionService.scala:64)
[error]         at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
[error]         at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:577)
[error]         at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
[error]         at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
[error]         at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
[error]         at java.base/java.lang.Thread.run(Thread.java:1623)
[error] Caused by: java.io.FileNotFoundException: https://repo.maven.apache.org/maven2/com/google/protobuf/protoc/3.10.0/protoc-3.10.0-osx-aarch_64.exe
[error]         at java.base/sun.net.www.protocol.http.HttpURLConnection.getInputStream0(HttpURLConnection.java:1976)
[error]         at java.base/sun.net.www.protocol.http.HttpURLConnection.getInputStream(HttpURLConnection.java:1581)
[error]         at java.base/sun.net.www.protocol.https.HttpsURLConnectionImpl.getInputStream(HttpsURLConnectionImpl.java:224)
[error]         at com.github.os72.protocjar.Protoc.downloadFile(Protoc.java:333)
[error]         at com.github.os72.protocjar.Protoc.downloadProtoc(Protoc.java:288)
[error]         at com.github.os72.protocjar.Protoc.extractProtoc(Protoc.java:220)
[error]         at com.github.os72.protocjar.Protoc.extractProtoc(Protoc.java:184)
[error]         at com.github.os72.protocjar.Protoc.runProtoc(Protoc.java:68)
[error]         at com.github.os72.protocjar.Protoc.runProtoc(Protoc.java:55)
[error]         at sbtprotobuf.ScopedProtobufPlugin.$anonfun$projectSettings$9(ProtobufPlugin.scala:67)
[error]         at sbtprotobuf.ScopedProtobufPlugin.$anonfun$projectSettings$9$adapted(ProtobufPlugin.scala:67)
[error]         at sbtprotobuf.ScopedProtobufPlugin.executeProtoc(ProtobufPlugin.scala:110)
[error]         at sbtprotobuf.ScopedProtobufPlugin.compile(ProtobufPlugin.scala:128)
[error]         at sbtprotobuf.ScopedProtobufPlugin.$anonfun$sourceGeneratorTask$3(ProtobufPlugin.scala:170)
[error]         at sbt.util.FileFunction$.$anonfun$cached$1(FileFunction.scala:80)
[error]         at sbt.util.FileFunction$.$anonfun$cached$4(FileFunction.scala:153)
[error]         at sbt.util.Difference.apply(Tracked.scala:414)
[error]         at sbt.util.Difference.apply(Tracked.scala:394)
[error]         at sbt.util.FileFunction$.$anonfun$cached$3(FileFunction.scala:149)
[error]         at sbt.util.Difference.apply(Tracked.scala:414)
[error]         at sbt.util.Difference.apply(Tracked.scala:389)
[error]         at sbt.util.FileFunction$.$anonfun$cached$2(FileFunction.scala:148)
[error]         at sbtprotobuf.ScopedProtobufPlugin.$anonfun$sourceGeneratorTask$1(ProtobufPlugin.scala:172)
[error]         at scala.Function1.$anonfun$compose$1(Function1.scala:49)
[error]         at sbt.internal.util.$tilde$greater.$anonfun$$u2219$1(TypeFunctions.scala:62)
[error]         at sbt.std.Transform$$anon$4.work(Transform.scala:68)
[error]         at sbt.Execute.$anonfun$submit$2(Execute.scala:282)
[error]         at sbt.internal.util.ErrorHandling$.wideConvert(ErrorHandling.scala:23)
[error]         at sbt.Execute.work(Execute.scala:291)
[error]         at sbt.Execute.$anonfun$submit$1(Execute.scala:282)
[error]         at sbt.ConcurrentRestrictions$$anon$4.$anonfun$submitValid$1(ConcurrentRestrictions.scala:265)
[error]         at sbt.CompletionService$$anon$2.call(CompletionService.scala:64)
[error]         at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
[error]         at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:577)
[error]         at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
[error]         at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
[error]         at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
[error]         at java.base/java.lang.Thread.run(Thread.java:1623)
[error] (protoBindings / Protobuf / protobufGenerate) error occurred while compiling protobuf files: https://repo.maven.apache.org/maven2/com/google/protobuf/protoc/3.10.0/protoc-3.10.0-osx-aarch_64.exe
[error] Total time: 25 s, completed 14 Jun 2023, 07:28:04
```

--- a/proto-bindings/build.sbt
+++ b/proto-bindings/build.sbt
@@ -6,7 +6,7 @@ enablePlugins(ProtobufPlugin)
 val protocLocalDir   = "protoc"
 val protocBinaryPath = s"$protocLocalDir/bin/protoc"
 ProtobufConfig / protobufProtoc := protocBinaryPath
-ProtobufConfig / version        := "3.10.0"
+ProtobufConfig / version        := "3.18.0"
 ProtobufConfig / sourceDirectories += (ProtobufConfig / protobufExternalIncludePath).value
 ProtobufConfig / protobufGenerate := (ProtobufConfig / protobufGenerate)
   .dependsOn(copyLatestCpgProto)

